packages/sdks/snippets/qwik-city/src/routes/live-preview/[...index]/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  component$,
+  useSignal,
+  useTask$,
+  useVisibleTask$,
+} from '@builder.io/qwik';
+import { routeLoader$ } from '@builder.io/qwik-city';
+import type { BuilderContent } from '@builder.io/sdk-qwik';
+import {
+  fetchOneEntry,
+  getBuilderSearchParams,
+  subscribeToEditor,
+} from '@builder.io/sdk-qwik';
+
+export const useBuilderContent = routeLoader$(async ({ url }) => {
+  const data = await fetchOneEntry({
+    model: 'blog-data',
+    apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+    options: getBuilderSearchParams(url.searchParams),
+    userAttributes: {
+      urlPath: url.pathname,
+    },
+  });
+  return data ?? null;
```

Comment:
I want to avoid `?? null` and `|| null` whenever possible as they are unnecessary.

```suggestion
  return data;
```

Reply:
Noted. Will keep it in mind for future PRs as well.